react-app/src/components/RoomsList.js

```diff
@@ -0,0 +1,37 @@
+import React from 'react'
+import axios from 'axios'
+import Room from './Room'
+
+const getRoomUrl = 'https://hall-booking-system.herokuapp.com/room';
+
+class RoomsList extends React.Component {
+  constructor() {
+    super()
+    this.state = {
+      rooms: [],
+    };
+  }
+
+  componentDidMount() {
+    axios.get(getRoomUrl)
+    .then((response) => {
+      if(response.status === 500) {
+        return
+      }
+      this.setState({
+        rooms: response.data.rooms
+      })
+  });
```

Comment:
add a `.catch(error)` to catch if there are errors
in this case do a `console.error(error)`